user/serializers.py

```diff
@@ -2,0 +1,29 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+
+
+class UserSerializer(serializers.ModelSerializer):
+    def validate(self, attrs):
+        data = super(UserSerializer, self).validate(attrs)
+        get_user_model().validate_password(
+            attrs["password"],
+            serializers.ValidationError
+        )
+        return data
+
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
```

Comment:
The 'min_length' option in extra_kwargs doesn't actually enforce a minimum length for the password. You need to enforce this in the validate method.